src/Grid/utils.jl

```diff
@@ -1,348 +1,3 @@
-"""
```

Comment:
Where did this file go?

Reply:
Replaced by code in [grid.jl](https://github.com/Ferrite-FEM/Ferrite.jl/blob/6fc8fc479f381989585847b3b9bba282d044347b/src/Grid/grid.jl#L676-L716)

Could make sense to move all functionality relating to adding / creating sets in the grid to utils. Can do that before merging.

Ref for motivation: *Extract Boundary entities* in https://github.com/Ferrite-FEM/Ferrite.jl/pull/914#issue-2295919300 and discussions https://github.com/Ferrite-FEM/Ferrite.jl/pull/914#issuecomment-2111881613 -> https://github.com/Ferrite-FEM/Ferrite.jl/pull/914#issuecomment-2112837346